### ISSUE TITLE
RDP_SCALE in winapps.conf breaking xfreerdp command

### ISSUE THREAD

**Issue by adil192** [Fmstrat/winapps#195]
When my winapps.conf file is this (with an RDP_SCALE set)...
```python
RDP_USER="..."
RDP_PASS="..."
#RDP_DOMAIN="MYDOMAIN"
#RDP_IP="192.168.123.111"
RDP_SCALE=200
#RDP_FLAGS="/audio-mode:1"
#MULTIMON="true"
#DEBUG="true"
```
...`bin/winapps check` fails and prints how to use the xfreerdp command:
```
FreeRDP - A Free Remote Desktop Protocol Implementation
See www.freerdp.com for more information

Usage: xfreerdp [file] [options] [/v:<server>[:port]]

Syntax:
    /flag (enables flag)
    /option:<value> (specifies option with value)
    +toggle -toggle (enables or disables toggle, where '/' is a synonym of '+')
```

#  

This doesn't happen when I comment out the `RDP_SCALE` line.

**Comment by avanishsubbiah** [comment]
I have the exact same issue with RDP_SCALE breaking the xfreerdp command.

**Comment by avanishsubbiah** [comment]
Here's a quick script that you can use to implement the scaling automatically. You need to link this to /usr/bin/ and move the actual xfreerdp application somewhere else that isn't on you device path (like your home directory), and provide the script with the device path of the original xfreerdp application. Afterwards all calls to "xfreerdp" will run the script instead and pass all arguments along with the script's set argument for `/scale-desktop:125`. You can change the value on the scale-desktop to whatever percentage you want or change the arguments passed in entirely if you want some other default arguments passed in.

```
#!/bin/bash
[INSERT PATH TO xfreerdp APPLICATION HERE MUST NOT BE "/usr/bin)"] /xfreerdp "${@:1}" /scale-desktop:125
```

**Comment by adil192** [comment]
> ```
> #!/bin/bash
> [INSERT PATH TO xfreerdp APPLICATION HERE MUST NOT BE "/usr/bin)"] /xfreerdp "${@:1}" /scale-desktop:125
> ```

Hey I've tried to incorporate that into the actual project. I don't really work with bash scripts (my first pull request) so please can you double check the commit I've just added if possible?

**Comment by Amarjit3** [comment]
> > ```
> > #!/bin/bash
> > [INSERT PATH TO xfreerdp APPLICATION HERE MUST NOT BE "/usr/bin)"] /xfreerdp "${@:1}" /scale-desktop:125
> > ```
> 
> Hey I've tried to incorporate that into the actual project. I don't really work with bash scripts (my first pull request) so please can you double check the commit I've just added if possible?

Luckily I've discovered the [solution](https://github.com/Fmstrat/winapps/pull/216) - someone has already done the work but the fix hasn't really been documented.

`git clone https://github.com/adil192/winapps.git
bin/winapps check
./installer.sh`

The file located in `~/.config/winapps/winapps.conf` should have `RDP_SCALE=125` and then all works.

**Comment by adil192** [comment]
> Luckily I've discovered the [solution](https://github.com/Fmstrat/winapps/pull/216) - someone has already done the work but the fix hasn't really been documented.
> `git clone https://github.com/adil192/winapps.git`

Yep that's my fork, glad you found it helpful. What issues did you have with the way it's documented?
From https://github.com/adil192/winapps#step-3-creating-your-winapps-configuration-file:

>You will need to create a ~/.config/winapps/winapps.conf configuration file with the following information in it:
>
> ```
> RDP_USER="MyWindowsUser"
> RDP_PASS="MyWindowsPassword"
> #RDP_DOMAIN="MYDOMAIN"
> #RDP_IP="192.168.123.111"
> #RDP_SCALE=100
> #RDP_DEVICE_SCALE=100 # can only be 100|140|180
> #RDP_FLAGS=""
> #MULTIMON="true"
> #DEBUG="true"
> ```
> 
> The username and password should be a full user account and password, such as the one created when setting up Windows or a domain user. It cannot be a user/PIN combination as those are not valid for RDP access.
> 
> Options:
> - ...
> - On high-resolution (UHD) displays, you can set RDP_SCALE to the scale you would like. RDP_DEVICE_SCALE is automatically set depending on your RDP_SCALE, but you can manually specify one of [100|140|180] if you wish.

**Comment by acidburnmonkey** [comment]
can somebody clarify what is the deference on setting RDP_SCALE and RDP_DEVICE_SCALE ?
